config/config.go

```diff
@@ -63,6 +63,7 @@ type HTTPConfig struct {
 	Port         int    `mapstructure:"port"`
 	UseAuth      bool   `mapstructure:"use_auth"`
 	AuthToken    string `mapstructure:"auth_token"`
+	EnablePprof  bool   `mapstructure:"enable_pprof"`
```

Comment:
### minor

Maybe it would be better to have the same config name as in bux.
```
	// DebugProfiling is a flag for enabling additional endpoits for profiling with use of pprof.
	DebugProfiling bool `json:"debug_profiling" mapstructure:"debug_profiling"`
```

Reply:
I didn't realize it is in HttpConfig. Therefore, maybe EnablePprof is better name. Or maybe `ProfilingEndpointsEnabled`. I'm leaving this up to you.

Just add a documentation comment what is it for. IMHO actually it would be good to have those comments on all options, as this is open source project, but it is out of the scope of your task.

Reply:
Added comments for all options except functions types